### PR TITLE
embotics-vcommander-plugin

### PR DESCRIPTION
**Pull request by aelliotatemboticsdotcom**
# Description

https://github.com/jenkinsci/embotics-vcommander-plugin
https://issues.jenkins-ci.org/projects/HOSTING/issues/HOSTING-555

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by aelliotatemboticsdotcom** [comment]
needs label needs-authorization

**Comment by aelliotatemboticsdotcom** [comment]
@orrc  or @daniel-beck can you approve or forward to another maintainer? Thank you!

**Comment by aelliotatemboticsdotcom** [comment]
I'm the only maintainer for this project so far so I couldn't @ another user.

**Comment by aelliotatemboticsdotcom** [comment]
@oleg-nenashev can you approve this?  looking for a maintainer to look it over as I'm the sole committer.

**Comment by daniel-beck** [comment]
No Artifactory user for 'aelliotatemboticsdotcom'. Please review this repo's readme and the full PR template to understand how to address this.

**Comment by aelliotatemboticsdotcom** [comment]
@daniel-beck login completed for both accounts listed.

**Comment by daniel-beck** [comment]
`path` is the effective path to the plugin in the Maven repo from Maven coordinates. See this repo's README for an example.

**Comment by aelliotatemboticsdotcom** [comment]
@daniel-beck I think I have it fixed now?  Sorry for the headaches, first-timer here.

**Comment by aelliotatemboticsdotcom** [comment]
@daniel-beck what's wrong here?? I've compared the pom and permissions file to others which have been merged and I don't see what the issue is....can you comment?  I've double-checked everything against the readme instructions and can't see what the fix should be.  help?  or can you specify what the actual problem is?

**Comment by daniel-beck** [comment]
The file name is wrong (`plugin-` is a prefix there, not a suffix), and we don't support new plugins with the `-plugin` suffix. See https://jenkins.io/doc/developer/publishing/style-guides/ for naming conventions.

File: permissions/embotics-vcommander-plugin.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "embotics-vcommander-plugin"
```
[the remark this answers is not in the document]
Comment (aelliotatemboticsdotcom) [reply permissions/embotics-vcommander-plugin.yml]: fixed

**Comment by aelliotatemboticsdotcom** [comment]
thanks for the pointers on this, it's confusing for a first-timer.

**Comment by batmat** [comment]
The file is still wrong, should be prefixed.
And now you added a file to the PR, not removed the previous one. But neither of them are correct.

Maybe compare with the few examples you have along your own file in the `permissions` directory on your machine @aelliotatemboticsdotcom ?

**Comment by batmat** [comment]
Really @aelliotatemboticsdotcom we're trying to help, but please read what we write and make sure you address **all** the steps/things we say.

Quoting myself:
> And now you added a file to the PR, not removed the previous one. 

There should be one file only...

**Comment by aelliotatemboticsdotcom** [comment]
can we get this checked again? sorry for the issues.

**Comment by aelliotatemboticsdotcom** [comment]
@daniel-beck can you review this again? thanks so much.

**Pull request merged**